bedrock/mozorg/templates/mozorg/contribute/studentambassadors/landing.html

```diff
@@ -51,7 +51,7 @@
         <section id="sec-what">
           <h2>{{ _('What We Do') }}</h2>
           <ul>
-            <li>{{ _('<a href="%s">Promote Firefox</a> for Desktop and Android')|format(url('mozorg.contribute.friends')) }}</li>
+            <li>{{ _('<a href="%s">Support Firefox</a> for Desktop and Android')|format(url('mozorg.contribute.friends')) }}</li>
```

Comment:
Since we're editing this line anyway, should we add iOS? "Support Firefox for desktop, Android and iOS" ?